src/test/java/com/plaid/client/integration/TransactionsGetTest.java

```diff
@@ -32,12 +32,18 @@
   @Before
   public void setUp() throws Exception {
     Response<SandboxPublicTokenCreateResponse> createResponse =
-      client().service().sandboxPublicTokenCreate(new SandboxPublicTokenCreateRequest(TARTAN_BANK_INSTITUTION_ID, Arrays.asList(Product.TRANSACTIONS))).execute();
+      client().service()
+        .sandboxPublicTokenCreate(new SandboxPublicTokenCreateRequest(TARTAN_BANK_INSTITUTION_ID,
+          Collections.singletonList(Product.TRANSACTIONS)))
```

Comment:
nonblocking nit: this is kind of confusing which parentheses match with which... can we have something that looks like this (feel free to call me out if this is not Java convention):

```
        .sandboxPublicTokenCreate(new SandboxPublicTokenCreateRequest(
          TARTAN_BANK_INSTITUTION_ID,
          Collections.singletonList(Product.TRANSACTIONS)
        ))
```

Reply:
java is basically wildwest when it comes to stuff like this. I use https://github.com/square/java-code-styles which is used by Jake Wharton of android fame